config/schema/media_entity_embeddable_video.schema.yml

```diff
@@ -17,6 +17,13 @@ media_entity_embeddable_video.settings:
     wochit_progn:
       type: string
       label: 'Publisher information used by Wochit ("progn" argument in embed code)'
+    youtube:
+      type: mapping
+      label: 'YouTube'
+      mapping:
+        api_key:
+          type: string
```

Comment:
I think the data-type in entity schema and default value in media_entity_embeddable_video.settings does not match.